packages/dara-core/dara/core/internal/routing.py

```diff
@@ -332,6 +332,7 @@ async def get_data_variable(
                     store,
                     body.filters,
                     Pagination(offset=offset, limit=limit, orderBy=order_by, index=index),
+                    True,
```

Comment:
Could you pass the boolean as a kwarg in both places, would be much more readable than having to look up what this `True` means here